src/routes/(auth)/signout/+page.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+    import { backendService } from "$lib/grpc-api";
+    import { goto } from "$app/navigation";
+    import { onMount } from "svelte";
+
+    onMount(async () => {
+        try {
+            await backendService.logout({});
+        } catch (e) {
+            console.error("There was an error logging out: ", e);
+        }
+        goto("/signin");
```

Comment:
```suggestion
        } finally {
            goto("/signin", { replaceState: true });
        }
```

Maybe add this option to goto. If I understand this correctly, with this option, this page will not be added to the browser history or at least it will not be visited if you click on the back and forward buttons in the top left. I hope you know what I mean.

As well as moving this into a finally block, since it is literally for this use case.

Reply:
Great suggestion :+1: